OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -270,6 +270,16 @@ internal static void UpgradeActorRules(ModData modData, int engineVersion, ref L
 					}
 				}
 
+				if (engineVersion < 20160717)
```

Comment:
Should have bumped this during the rebase, sorry for not pointing it out yesterday. :S